Sources/Classes/ViewController/ScannerViewControllerDelegate.swift

```diff
@@ -1,5 +1,5 @@
 import UIKit
 
-public protocol ScannerViewControllerDelegate: AnyObject {
+@objc public protocol ScannerViewControllerDelegate: AnyObject {
```

Comment:
Can you remove the `: AnyObject`?

Since `@objc` protocols must be implemented by `NSObject` subclasses, we don't need this annotation.

Reply:
```suggestion
@objc public protocol ScannerViewControllerDelegate {
```